src/borg/archiver.py

```diff
@@ -4248,18 +4248,19 @@ def define_borg_mount(parser):
 
         1. Ask you to come up with a passphrase.
         2. Create a borg key (which contains some random secrets. See :ref:`key_files`).
-        3. Encrypt the key with your passphrase.
-        4. Store the encrypted borg key inside the repository directory (in the repo config).
+        3. Derive a "key encryption key" from your passphrase
+        4. Encrypt and sign the key with the key encryption key
```

Comment:
that sounds like "encrypt-and-mac" and using only 1 key for both (what we do for pbkdf2, but not any more for argon2).